src/calculateRentalCost.js

```diff
@@ -5,6 +5,10 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const rentCost = 40;
+  const rentOff = days < 3 ? 0 : days >= 3 && days >= 7 ? 50 : 20;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md): "Don't use literals in expressions (HARDCODE). Instead use constants explaining this values"